.github/workflows/_test_upstream_t5x.yaml

```diff
@@ -28,6 +28,11 @@ on:
         description: 'Name of the framework being used'
         required: false
         default: 'upstream-t5x'
+      ENABLE_FMHA:
```

Comment:
Is it okay that fmha is tested on demand since the default = 0?

If you'd rather it be part of the test suite, then here's a reference for how rosetta t5x tests look: 
https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_test_t5x_rosetta.yaml#L38-L53

Basically we matrixify the args passed and the JOB_NAME is defined as part of the matrix.

Reply:
Since we dispatch ci workflow and fmha is relevant to xla I have disabled it by default and only enable it here: https://github.com/NVIDIA/JAX-Toolbox/blob/add_t5x_fmha_test/.github/workflows/_ci.yaml#L249 That way separate job with fmha in the name is triggered with the job matrix. Why should I matrixify the args in this case?